digital_asset_types/src/dapi/change_logs.rs

```diff
@@ -1,5 +1,6 @@
 use sea_orm::sea_query::Expr;
 use sea_orm::{DatabaseConnection, DbBackend};
+use log::debug;
```

Comment:
why not just use tracing package everywhere?

Reply:
Not necessary, can use the log facade.